src/main/java/org/mule/extension/oauth2/api/tokenmanager/TokenManagerConfig.java

```diff
@@ -43,7 +45,7 @@
 @TypeDsl(allowTopLevelDefinition = true)
 @NoExtend
 @NoInstantiate
-public class TokenManagerConfig implements Lifecycle {
+public class TokenManagerConfig<Ctx extends ResourceOwnerOAuthContext & Serializable> implements Lifecycle {
```

Comment:
Java conventions dictate that generics need to be all caps. Also, why not simply making ResourceOwnerOAuthContext implement Serializable? it's a `@NoImplement` interface anyway.

Reply:
Even if i make `ResourceOwnerOAuthContext` serializable, that won't be seen by this code since this compiles against the older version